ci/builders/README.md

```diff
@@ -436,4 +432,60 @@ The migration to use the build definition language depends on the release | vali
 
 
 *   A change in the engine repository to remove the old build configurations from .ci.yaml file and updating the artifact destination in the builds using the build definition language will be landed.
-*   A change to remove old recipes will be landed in the recipes repository. 
+*   A change to remove old recipes will be landed in the recipes repository.
+
+
+## Triaging global generators
+
+Global generators can run locally if all their sub-build dependencies are downloaded. This section explains how to triage a local generator.
+
+### Prerequisites (one time installation)
+
+
+#### Install CAS utility
+
+CAS client is required to download the sub-build artifacts. To install it in your machine run the following steps:
+
+```
+mkdir $HOME/tools
+```
+
+Create an ensure.txt file with the following content:
+```
+infra/tools/luci/cas/mac-amd64 rP7SUASC3hxbjEyHDYyjOW3dr8vNgBmb5FpdxY3eFwgC
+```
+
+Run the following command to download the dependency from the cipd servers:
+```
+cipd ensure -root . -ensure-file ensure.txt
+# Add $HOME/tools to path and your ~/.bashrc
+```
+
+#### Gclient engine checkout
+
+Create a gclient checkout following instructions from [Setting up the engine environment](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
+
+### Download sub-builds to the gclient checkout out folder
+
+CAS sub-build artifacts can be downloaded using information from a LUCI build. Using https://ci.chromium.org/p/flutter/builders/prod/Mac%20mac_ios_engine/2818 as an example the execution details from steps 13 - 17 show the commands to download the archives. `-dir` parameter needs to be updated to point to the relative or full path to the out folder in your gclient checkout.
+
+These are the commands to execute for our example build:   
+
+```
+pushd <gclient checkout>/src/out
```

Comment:
Can any of this be scripted? We can put scripts in the `ci/` directory of the `flutter/engine` repo.

Reply:
It is possible but may require some research. Opened https://github.com/flutter/flutter/issues/124685 to track the CLI potential features.